recipes/install.rb

```diff
@@ -92,6 +92,7 @@
   docroot docroot
   es_server node['kibana']['es_server']
   kibana_port node['kibana']['java_webserver_port']
-  template 'kibana-nginx_file.conf.erb'
+  template node['kibana']['nginx']['template']
```

Comment:
This change seems to only work for nginx (what happens when `default['kibana']['webserver']='apache'`?), and duplicates the template resource on line 58 in the same file.